users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -628,3 +629,138 @@ def run_mel_baseline():
         os.path.join(gs.ALIAS_AND_OUTPUT_SUBDIR, "report_mel_baseline.csv"),
         values=report.get_values(),
         template=report.get_template())
+
+def run_mel_audio_perturbation():
```

Comment:
Please (re)move the audio perturbation related experiment(s) from `run_test_mel()`.

Reply:
Done